### ISSUE TITLE
FCOS is too sensitive to batch size?

### ISSUE THREAD

**Issue by karlind** [open-mmlab/mmdetection#1178]
I use FCOS to train my own dataset, but I find FCOS is sensitive to batch size.
Performance for different batch size may differ to almost 10%.
I tried to adjust learning rate to fit batch size(linear to batch size), but it don't help. Is there any clue for this?

**Comment by BlueBlueFF** [comment]
some probelm。。。。

**Comment by hellock** [comment]
It seems sensitive to the batch size. You may ask the authors.

**Comment by hyz-xmaster** [comment]
Hi, I think I have found the cause. Please see this bug fix commit: https://github.com/tianzhi0549/FCOS/commit/a62aeca5f31eaeda37cfaf7ab66d2e3d84abba32
